docs/overview.md

```diff
@@ -0,0 +1,64 @@
+# Interledger Overview
+
+Interledger enables the seamless exchange of value across different payment networks. Traditional payment networks 
+operate independently from each other. Sending value is easy only if the sender and recipient have accounts on the same 
+network, but it can be slow and expensive if they have accounts on different networks.
+
+Interledger is not tied to any one company, blockchain, or currency. You can easily send XRP to someone who wants to 
+receive ETH, or you can send USD through a Paypal account to someone who wants to receive USD in a Venmo account. 
+Interledger makes it easy to transact in whatever currency or payment network you choose.
+
+## What is Interledger?
+Interledger is a network of computers that enables sending value across independent payment networks. Similar to the Internet, 
+Interledger routes packets of value from source to destination. Computers on the Interledger network are 
+called *nodes*. Nodes can take one or more of the following roles: 
```

Comment:
There is a long and detailed debate about whether or not we want to call connectors nodes. I'm in favour but when this was last discussed we decided to stick with connectors.

@emschwartz @sharafian @mDuo13

Reply:
I think connector is more appropriate. We're already using the term and `node` isn't very descriptive

Reply:
I am a bit confused...
Is the discussion about Node vs. Connector or Router vs. Connector?

Reply:
Node vs. Connector

Reply:
@adrianhopebailie are you suggesting we revise this entire set of paragraphs to purposefully _not_ use the "node" analogy at all?

We could accomplish that with the following change:

```
Interledger routes packets of value from source to destination. Computers on the Interledger network can operate in one of three roles:

* Sender - ....
```

Reply:
To me, "node" includes senders, receivers, and routers/connectors.

I'm fine with this description but for clarity's sake I would also mention that "routers" are also referred to as "connectors" throughout some of the documentation.

Reply:
Added a note.
https://github.com/interledger/interledger.github.io/pull/113/files#diff-f3b56000093113bd3bfb6c9c05e7e945R22

Reply:
Anywhere I want to refer to sender/router/receiver collectively, `node` seems to fit the best. For example, in the sentence - "Each ILP packet contains transaction information, which is private to the participating nodes". 
I don't know what else to call them. Does anybody have a better suggestion?

Reply:
+1 to what @emschwartz - I think when we refer to a collective of roles (e.g., sender/router/receiver) it's fine to use the term `node`.